packages/contractkit/src/identity/claims/verify.ts

```diff
@@ -18,13 +19,14 @@ import { ClaimTypes } from './types'
 export async function verifyClaim(
   claim: Claim,
   address: string,
-  metadataURLGetter: MetadataURLGetter
+  metadataURLGetter: MetadataURLGetter,
+  kit: ContractKit
```

Comment:
metadataURLGetter was basically me trying to avoid having to pass kit, but since we do that, we could get rid of metadataURLGetter?